install/kubernetes/cilium/values.yaml

```diff
@@ -16,6 +16,9 @@ config:
 operator:
   enabled: true
 
+  # Number of replicas to run for cilium-operator deployment.
+  numReplicas: 2
```

Comment:
Should we default this to 1 but allow override, given how late in the cycle this is for v1.7?